recipes/lmdb/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd libraries/liblmdb/
+export DESTDIR=$PREFIX
```

Comment:
This is how lmdb has you set the install prefix. Though it tacks on `/usr/local`, we have patched that out as mentioned above. So, this ensures everything will get installed in the conda environment.